src/devices/Common/Interop/Unix/Interop.X11.cs

```diff
@@ -0,0 +1,429 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```

Comment:
same question, why do we need X11 interop added as part of image library refactoring?